main.py

```diff
@@ -0,0 +1,16 @@
+from fastapi import FastAPI
+from sqlalchemy.orm import Session
+
+from City_CRUD_API import router as city_router
+from Temperature_API import router as temperature_router
+from database import SessionLocal
```

Comment:
The import of `SessionLocal` from `database` is not used in this file. Consider removing it to clean up the code and avoid confusion.